### PR TITLE
feat: add fastlane release scripts

### PR DESCRIPTION
**Pull request by aswin-s**
## Description

This PR adds fastlane release scripts for generating and publishing release builds for ios and android via fastlane

## Change log

- Added fastlane config for android project
- Added release scripts

## How to test

Below commands should work properly

```bash
yarn cli release android dev
```
```bash
yarn cli release android prod
```

TODO:

- version number is currently hardcoded to `v0.1.0`
- configure github api to fetch releases and auto increment versions

### CODE REVIEW

File: devops/src/cli/release.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  IOS = 'ios',
+}
+
+export enum Environment {
+  PROD = 'prod',
+  DEV = 'dev',
+}
+
+// TODO: Specify Github config
+const config = {
```
Comment (aswin-s) [inline devops/src/cli/release.ts:17]: Provide github config

File: devops/src/cli/release.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      chalk`{red Error: Uncommitted changes are present. Please commit changes before deployment }`,
+    )
+    return
+  }
+
+  process.chdir(getFrontendRootDir())
+  await runCommand(commands.yarn.setEnv(env))
+
+  // TODO getAppVersion after setting up versioning
+  // const appVersion = await getAppVersion(env)
```
Comment (aswin-s) [inline devops/src/cli/release.ts:69]: Uncomment after setting up versioning

File: mobile/android/fastlane/Fastfile [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  desc "Build and Submit App to Playstore"
+  lane :deploy do |options|
+   
+    
+    # UI.important "Current application version: " + options[:version]
```
Comment (aswin-s) [inline mobile/android/fastlane/Fastfile:33]: Uncomment these line after publishing initial version to playstore

**Comment by rintoj** [comment]
Merged

**Pull request closed without merging**